modules/appeals_api/spec/lib/stats_report_spec.rb

```diff
@@ -26,6 +27,7 @@ def create_transition_groups(group_size: 1, num_groups: described_class::STATUS_
   let(:outside_range_groups) { create_transition_groups }
 
   before do
+    allow(FeatureFlipper).to receive(:send_email?).and_return(false)
```

Comment:
This unrelated job calls `update_status!`, so it triggers the appeal received job for a few records. This `FeatureFlipper` setup stops the job from actually running and trying to send email.